sdk.mjs

```diff
@@ -116,43 +116,45 @@ export const titleToHtmlFriendly = (brandTitle) =>
  */
 export const htmlFriendlyToTitle = (htmlFriendlyTitle) =>
   htmlFriendlyTitle
-    .replace(/&#([0-9]+);/g, (_, num) => String.fromCodePoint(parseInt(num)))
-    .replace(
+    .replaceAll(/&#(\d+);/g, (_, number_) =>
+      String.fromCodePoint(Number.parseInt(number_, 10)),
+    )
+    .replaceAll(
       /&(quot|amp|lt|gt);/g,
-      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' })[ref],
+      (_, reference) => ({quot: '"', amp: '&', lt: '<', gt: '>'})[reference],
     );
 
 /**
  * Get path of *_data/simpe-icons.json*.
```

Comment:
```suggestion
 * Get path of *_data/simple-icons.json*.
```
Should we fix this little typo in this PR lol?

Reply:
Do you know about a mispell fixer like [mispell-fixer](https://github.com/vlajos/misspell-fixer) or [codespell](https://pypi.org/project/codespell/) for JS?

Reply:
I would avoid cspell for our project. You will need to maintain a super-long dictionary like this https://github.com/netless-io/flat/blob/main/cspell.config.js.

You can install it in your local environment.

Typos are OK for me. Just fix it when you find it.

Reply:
You could fix those typos in another PR. Those typos are not related to this topic.